network/load-balancer/concepts.mdx

```diff
@@ -135,6 +135,11 @@ Proxy protocol is an internet protocol used to transfer connection information f
 
 ## Sticky session
 
-A sticky session enables the Load Balancer to bind a user’s session to a specific Instance. This ensures that all subsequent sessions from the user are sent to the same Instance, while there is at least one active session.
+A sticky session enables the Load Balancer to bind a user’s session to a specific Instance. This ensures that all subsequent sessions from the user are sent to the same Instance, while there is at least one active session. Sticky sessions can be **cookie based** or **table based**. 
+
+- **Cookie-based** stickiness uses HTTP cookie value. The name of the cookie can be customized. The cookie is set by the Load Balancer on the first request, and subsequent requests using this cookie will land on the same backend.
```

Comment:
I would say that: Cookie-based stickiness uses an HTTP cookie to identify a session and the server in the backend pool it will stick to. For each incoming request without stickiness cookie, the load balancer creates a stickiness cookie and selects a backend for the session. Subsequent requests providing this cookie will land on the same backend. The name of the cookie can be customized.